Taskfile.yaml

```diff
@@ -201,3 +161,9 @@ tasks:
   kind-delete:
     cmds:
       - kind delete cluster --name {{.KIND_CLUSTERNAME}}
+
+  sync:charts:
```

Comment:
Rather than have this target a relative directory could we perform this sync to a git SHA so there's no change of unexpected changes making their way in?

Git cloning into a subdirectory or downloading the tar.gz of the repo at the given SHA are the first options that come to mind.

Reply:
Ok I will test the tar.gz route